src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -31,9 +40,12 @@ public void lowerPiston() {
         shooterSolenoid.set(false);
     }
 
-    /** @param shooterSpeed Velocity of shooter motors in RPM */
     public void shoot(double shooterSpeed) {
-        // shooterController.setReference(shooterSpeed, ControlType.kVelocity);
-        shooterMotor1.set(shooterSpeed);
+        shooterController.setReference(2000, ControlType.kVelocity);
+        // shooterMotor1.set(shooterSpeed);
+    }
```

Comment:
Maybe have two methods, one that takes  a speed and one that doesn't so you can use velocity vs voltage to control

Reply:
Indeed. We'll probably make this change next time. I wish there was a `tunable` for Java wpilib...